src/app/containers/Ad/Amp/index.test.jsx

```diff
@@ -41,160 +38,122 @@ describe('AMP Ads', () => {
-      adWithContext('leaderboard', toggleContextMock),
+      adWithContext('leaderboard'),
     );
 
     shouldMatchSnapshot(
       'should correctly render an AMP mpu ad',
-      adWithContext('mpu', toggleContextMock),
+      adWithContext('mpu'),
     );
   });
 
   describe('Assertions', () => {
-    describe('When ampAds is toggled on', () => {
-      const toggleState = {
-        ampAds: {
-          enabled: true,
-        },
-      };
-
-      const mockToggleDispatch = jest.fn();
-
-      const toggleContextMock = {
-        toggleState,
-        toggleDispatch: mockToggleDispatch,
-      };
-
-      it('should render two leaderboard ads', () => {
-        const { container } = render(
-          adWithContext('leaderboard', toggleContextMock),
-        );
-        const ampAd = container.querySelectorAll('amp-ad');
-
-        expect(ampAd.length).toBe(2);
-      });
+    it('should not render ad placeholder in UK', () => {
+      const { getByLabelText } = render(
+        <div className="amp-geo-group-eea amp-geo-group-gbOrUnknown">
+          {adWithContext('leaderboard')}
+        </div>,
+      );
+
+      expect(getByLabelText('Publicités')).not.toBeVisible();
+    });
 
-      it('should display leaderboard ad with values for all of the needed attributes', () => {
-        const { container } = render(
-          adWithContext('leaderboard', toggleContextMock),
+    it.each`
+      location           | className
+      ${'europe'}        | ${'amp-geo-group-eea'}
+      ${'rest of world'} | ${''}
+    `(
+      'should render ad placeholder outside of the UK - $location',
+      ({ className }) => {
+        const { getByLabelText } = render(
+          <div className={className}>{adWithContext('leaderboard')}</div>,
         );
 
-        const ampAd = container.querySelectorAll('amp-ad');
-        ampAd.forEach(ad => {
-          expect(ad).toHaveAttribute('data-block-on-consent', 'default');
-          expect(ad).toHaveAttribute('data-npa-on-unknown-consent', 'true');
-          expect(ad).toHaveAttribute('media');
-          expect(ad).toHaveAttribute('type');
-          expect(ad).toHaveAttribute('width');
-          expect(ad).toHaveAttribute('height');
-          expect(ad).toHaveAttribute('data-multi-size');
-          expect(ad).toHaveAttribute('data-slot');
-          expect(ad).toHaveAttribute('data-amp-slot-index');
-          expect(ad).toHaveAttribute('data-a4a-upgrade-type');
-          expect(ad).toHaveAttribute(
-            'json',
-            JSON.stringify(adJsonAttributes('leaderboard')),
-          );
-        });
-      });
-
-      it('should render one mpu ad', () => {
-        const { container } = render(adWithContext('mpu', toggleContextMock));
-        const ampAd = container.querySelectorAll('amp-ad');
+        expect(getByLabelText('Publicités')).toBeVisible();
+      },
+    );
 
-        expect(ampAd.length).toBe(1);
-      });
+    it('should render two leaderboard ads', () => {
```

Comment:
Is this correct? do we render two leader board ads? I thought it was a single leader board and then an MPU slot?

Reply:
hmm this is an existing test that has been moved, I can take a look

Reply:
ahh ok its probably just a misleading test label then. we do expect two ads to render, just not two leaderboards

Reply:
Looks to me like it relates to this:
https://github.com/bbc/simorgh/blob/latest/src/app/containers/Ad/Amp/AdSlot.jsx#L64..L79

It renders a slot for mobile and desktop respectively, presumably with suitable logic to make the right one visible at different breakpoints?

Reply:
That's correct. I remember leaving a comment at the time asking if it's really neccessary to support desktop ad slot sizes on AMP pages, I pulled some numbers in July 2020 (!), and 0.69% of AMP traffic was using a browser width of above 720px.

Reply:
wow, thats a pretty low number. I'd suggest we leave this for now but have @aeroplanejane investigate this in the future as we might be able to simply the setup